frontend_lib/src/component/Timeline/Timeline.jsx

```diff
@@ -88,7 +88,10 @@ export class Timeline extends React.Component {
                     customClass={props.customClass}
                     customColor={props.customColor}
                     apiUrl={props.apiUrl}
+                    contentId={Number(content.content_id)}
+                    workspaceId={Number(props.workspaceId)}
                     author={content.author}
+                    loggedUserId={Number(props.loggedUser.userId)}
```

Comment:
Don't forget to add those props in `propTypes`.

Reply:
Done there, or are you speaking about something else?

https://github.com/tracim/tracim/pull/4278/files/b1ff3d0a12fa44efa1d43190feb8a64f6a33506e..4c4e1a329a3060a0e1137ed0681e180ac35564e7#diff-41ba796e6d8d6c3b7d2e3e9e8c8268d2d2226b0ed2ac20e93844bcb1a37de720R80-R82